kernel/arch/i386/boot/stivale2.S

```diff
@@ -18,16 +18,16 @@ __stivale2_header:
     dd 0 ; stack, high bits
     dq 0 ; flags
     ; tags
-    dd 0 ; __stivale2_framebuffer
+    dd __stivale2_framebuffer
     dd 0
 
-%if 0
+%if 1
 section .early_data
 align 4
 __stivale2_framebuffer:
     dq 0x3ecc1bc43d0f7971
     dq 0
-    dw 1280
-    dw 720
-    dw 32
+    dw 0
```

Comment:
I know we didn't do this before, but could you add comments saying what each of these fields are?